README.md

```diff
@@ -57,10 +57,23 @@ Fb::Auth.new(redirect_uri: redirect_uri, code: code).access_token
  # => "kefjej49s82hFS@2333233222FDh66"
 ```
 
+Fb::User#pages
+---------------------
+
+Once you have successfully obtain an access token, you can fetch the pages managed
+by that access token. Calling `Fb::User.new(access_token).pages' will return an
```

Comment:
@S1v4 There is a typo here but you can fix this in a later commit. 

After `.pages` you need a backtick.